src/main/resources/db/migration/courtcase/V2123__add_case_comments_table.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
+
+create index case_id_idx on court_case (case_id);
+
+CREATE TABLE CASE_COMMENTS (
+   ID               SERIAL      PRIMARY KEY,
+   CASE_ID          TEXT        NOT NULL,
+   COMMENT          TEXT        NOT NULL,
+   AUTHOR           TEXT        NOT NULL,
+   CREATED          TIMESTAMP   NOT NULL DEFAULT now(),
+   CREATED_BY       TEXT        NULL,
+   CREATED_BY_UUID  TEXT        NOT NULL,
```

Comment:
Could use the `UUID` type here. In theory the same applies for CASE_ID but there's an [outstanding bug](https://dsdmoj.atlassian.net/browse/PIC-2375) which means this won't fly just yet